config/initializers/react.rb

```diff
@@ -0,0 +1,3 @@
+Consul::Application.configure do
+  config.react.addons = true # defaults to false
```

Comment:
I started using an addon called `cloneWithProps` but it's deprecated favor to `cloneElement` method.